packages/package_velvet_1_2_10/tool_dependencies.xml

```diff
@@ -0,0 +1,31 @@
+<tool_dependency>
+    <package name="velvet" version="1.2.10">
+        <install version="1.0">
+            <actions>
+                <action type="download_by_url">https://github.com/dzerbino/velvet/archive/9adf09f7ded7fedaf6b0e5e4edf9f46602e263d3.zip</action>
```

Comment:
@nsoranzo is that the commit that corresponds to what can be downloaded from their website?

Otherwise, https://github.com/dzerbino/velvet/commit/5cea317b4ec5c3f0fd2931321bc7231d8a250682 is a good option since that's when the version bump occurred.